app/javascript/packs/admin/editUser.jsx

```diff
@@ -1,8 +1,42 @@
 import { initializeDropdown } from '@utilities/dropdownUtils';
 
-let preact;
-let AdminModal;
-const modalContents = new Map();
+function adjustCreditRange(event) {
+  const {
+    target: { value, name, form },
+  } = event;
+
+  if (name === 'user[credit_action]') {
+    const creditAmount = form['user[credit_amount]'];
```

Comment:
Form elements can be [accessed by their name attribute on a form element](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements).